src/utils/ant.test.ts

```diff
@@ -0,0 +1,70 @@
+import { strict as assert } from 'node:assert';
+import { describe, it } from 'node:test';
+
+import { sortANTRecords } from './ant.js';
+
+describe('sortANTRecordsByPriority', () => {
+  it('should sort records by priority and then lexicographically', () => {
+    const records = {
+      undername1: { priority: 1, transactionId: 'test', ttlSeconds: 1 },
```

Comment:
might be insightful to have undername11 with priority 1 to ensure the lexicographical sort is working as expected.

Reply:
And perhaps undername01

Reply:
added in https://github.com/ar-io/ar-io-sdk/pull/385